src/languages/cpp.js

```diff
@@ -122,7 +171,10 @@ export default function(hljs) {
     relevance: 0
   };
 
-  const FUNCTION_TITLE = regex.optional(NAMESPACE_RE) + hljs.IDENT_RE + '\\s*\\(';
+  const FUNCTION_TITLE = regex.either(
+    regex.optional(NAMESPACE_RE) + hljs.IDENT_RE + '\\s*\\(',
+    'operator\\s*' + OPERATOR_RE + '\\s*\\(',
```

Comment:
The string "operator" is not part of a function title... and if you are trying to highlight the operator itself as a `title.function` in these cases it'd be better to use a multi-matcher to do that... see the documentation for `beginMatch`... so we'd match:

`"operator" [ws] [operator]` and scope each segment differently.

Reply:
```cpp
struct test{
	int a,b;
	inline bool operator+(const test&kkk)const{return a+kkk.b;}
};
```

I mean to highlight "operator" because I've noticed this case. I think it's a part of function title.

Reply:
Did you try the tools/developer.html tool?  This already works without any additional 
changes as I can tell.

![Uploading Screen Shot 2024-01-20 at 12.22.53 PM.png…]()

Reply:
`operator` is detected as a keyword instead of title.

Reply:
OH... I see what you're expecting now... hmmm... let me noodle on that...

Reply:
I see the intent, but how is it you decided `operator` is part of the 'function name' in this case?  Is that a CPP way of thinking?  Does CPP not officially declare `operator` a keyword?

I think you can make the argument the operator (in this context) is indeed `title.function` scope since it's effectively the "name" of the function, but what leads you to suggest that "operator" is also part of the function name/title? (vs just being a keyword).

Reply:
What's your opinion? You mean "operator" is used to decorate the real operator?

Reply:
Oh I think you are right after reading smth on cppreference.

Reply:
So how to deal with the overloaded operator? `title` or `operator`?

Reply:
But there is no `<span class="hljs-function"></span>` now.

Reply:
Currently nothing is working properly :(